### DISCUSSION TITLE
Formula to capture AutoID from related table

### DISCUSSION THREAD

**Discussion by dmganly** [Budibase/budibase#6036] [Help] [answered]
I am trying to create a row in an InvoiceDetails table and need to automatically add the InvoiceID (or Auto ID) from the related Invoice table.  I understand this is possible using a formula, but could not find the syntax for this formula.   I guess this should be a common enough requirement for many applications.  
Any other suggestions or pointers towards creating an invoicing system.  Thanks.

**Comment by melohagan** [comment, marked as answer]
Hi @dmganly 

This should be fairly straightforward: https://docs.budibase.com/docs/formula#fields-from-relationships

If you scroll down the left hand pane when creating your formula, then you should be able to click on the related field you want which will inject the correct syntax:

![Screenshot 2022-05-24 at 08 58 46](https://user-images.githubusercontent.com/101575380/169980053-98727245-7d83-4331-bf6d-a21e16ac1bb5.png)

**Reply by dmganly** [reply]
Many thanks,  I'll get there.
